include/Typedef.h

```diff
@@ -22,6 +22,19 @@ typedef double real;
 typedef float  real;
 #endif
 
+#ifndef GRAMFE_ENABLE_SINGLE_PRECISION
+#  ifdef GRAMFE_FLOAT8
+#     error "Cannot enable GRAMFE_FLOAT8 and GRAMFE_ENABLE_SINGLE_PRECISION at the same time!"
+#  else
+#     define GRAMFE_FLOAT8
+#  endif
+#endif
```

Comment:
Right, but I could not think of a good way to disable `GRAMFE_FLOAT8` using the only the Makefile if we enable it by default. Of course, one could remove this option from the Makefile anyway, since making it stable will require modifications to the source code anyway.

Reply:
@KunkelAlexander Agree. For now, we don't want users to modify `GRAMFE_FLOAT8`. So how about the following?
- Remove `GRAMFE_ENABLE_SINGLE_PRECISION` from `Makefile` and throughout gamer.
- Move `GRAMFE_FLOAT8` from `Typedef.h` to `Macro.h` and enable it by default.